### ISSUE TITLE
Enums can only contain strings

### ISSUE THREAD

**Issue by jofaul** [Safe-DS/Library-Analyzer#39]
### Is your feature request related to a problem?

In some cases, enums with values of several types can be useful. This is currently impossible

### Desired solution

Enhance enum annotation to be able to choose between types of string/bool/number/none for each enum value.

### Additional Context (optional)

Examples:

- `#/sklearn/sklearn.preprocessing._data/normalize/axis`
